docker/app/000-default.conf

```diff
@@ -0,0 +1,35 @@
+ServerName 127.0.0.1
+
+<VirtualHost *:80>
+    DocumentRoot /var/www/web
+
+    <Directory /var/www/web>
+        Require all granted
+        DirectoryIndex app.php
```

Comment:
Like you want, but personally I prefer to have the production bootstrap script when the user request `http://127.0.0.1/` and manually specify `http://127.0.0.1/app_dev.php/` for development.